main.cpp

```diff
@@ -36,6 +37,23 @@ std::vector<std::string> initialize_combolist(std::stringstream& combo_names)
 	return combo_list;
 }
 
+void initialize_wordlist(std::string wordlist_path, std::vector<std::string>& words)
```

Comment:
`std::string const&` to avoid copy; also you can use `std::filesystem::path` instead here; return vector instead of out parameter...

Reply:
That's how it was originally but I wanted to just work with a single vector, will update